### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by vikipretium**
# Description

In this project, we built a basic website that allows users to add/remove books from a list. We used JavaScript objects and arrays to achieve the requirement.

### General requirements
 - There are no **linter** errors. :heavy_check_mark: 
 - Followed the **Github** flow. :heavy_check_mark: 
 - The documentation is professional. :heavy_check_mark: 

### HTML/CSS/JS requirements
 - We followed the list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
 - We followed the list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).
 ### Project Requirements

- Implemented a basic **UI** with plain **HTML**. :heavy_check_mark: 
- Created an **array** collection of objects that keeps a list of books. :heavy_check_mark: 
- Created a **function** to add a new book to the collection, with title and author. :heavy_check_mark: 
- Created a **function** to remove a book from the collection. :heavy_check_mark: 
- Displayed all books saved in the collection in the top part of the page. :heavy_check_mark: 
- Made sure that when a user clicks the **"Add"** button: A new book is added to the collection. The new book is displayed on the page. :heavy_check_mark: 
- Made sure that when a user clicks the **"Remove"** button: The correct book is removed from the collection. The correct book disappears from the page. :heavy_check_mark: 
- Made sure that data is preserved in the **browser's memory** by using [localStorage] :heavy_check_mark: 
- Used only **vanilla JavaScript**. :heavy_check_mark: 

Thank you for taking the time to review our project's pull request. :pray:

### CODE REVIEW

File: assets/js/book-manager.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    let books = this.getToloStorage();
+    const tempBooks = [];
+    books.forEach((book, index) => {
+      if (index !== parseInt(id, 10)) {
+        tempBooks.push(book);
+      }
+    });
+    books = tempBooks;
+    localStorage.setItem('books', JSON.stringify(books));
+  };
```
Comment (TedLivist) [inline assets/js/book-manager.js]: - [ ] There seems to be a bug in your app. I loaded your app and add some books, now when I delete books, they disappear from the page but not from the `localStorage` collection. This means that when I reload the page, the books I previously deleted still reappear. Kindly fix this.

  Books are added | One book deleted | Second book deleted
  :---:|:---:|:---:
  pasu-1st | pasu-2nd | pasu-3rd

File: README.md [hunk 2]
```diff
@@ -4,8 +4,7 @@
 
 > One paragraph statement about the project.
```
Comment (TedLivist) [inline README.md]: - [x] Your README has a title for Portfolio whereas this is a different project, kindly correct your title
- [ ] Kindly add a more descriptive paragraph about the project

File: README.md [hunk 3]
```diff
@@ -4,8 +4,7 @@
...
 
-![screenshot](./assets/images/Screenshot.png)
-![screenshot](./assets/images/Desktop.png)
+![screenshot]()
```
Comment (TedLivist) [inline README.md]: - [x] Kindly fix your screenshot image if you have one, or remove it entirely

File: README.md [hunk 4]
```diff
@@ -17,7 +16,7 @@ Additional description about the project and its features.
 
 ## Live Demo
 
-[Live Demo Link](https://vikipretium.github.io/portfolio_project/)
+[Live Demo Link](https://vikipretium.github.io/)
```
Comment (TedLivist) [inline README.md:93]: - [x] In your README, kindly add instructions on how users can set up and run your app locally.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @vikipretium and @mwafrika,

Good job so far!
✔️ Books are added to page and storage
✔️ Books are removed from the page
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
#### _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: assets/js/book-manager.js (hunk 1 above)
Comment (vikipretium) [reply assets/js/book-manager.js]: Hello @TedLivist,
Thanks for the feedback. We are working on it. But can you please try our deployment page?
[Live Link](https://vikipretium.github.io/awesome-books-project/)

File: README.md (hunk 2 above)
Comment (vikipretium) [reply README.md]: Okay @TedLivist I will fix it!

File: assets/js/book-manager.js (hunk 1 above)
Comment (TedLivist) [reply assets/js/book-manager.js]: Hi @vikipretium, the live link still exhibits the same behaviour I highlighted

File: README.md (hunk 4 above)
Comment (vikipretium) [reply README.md:93]: fixed it !

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi team,

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**